ansible/configs/ocp4-disconnected-osp-lab/env_vars.yml

```diff
@@ -0,0 +1,578 @@
+###### VARIABLES YOU SHOULD CONFIGURE FOR YOUR DEPLOYEMNT
+###### OR PASS as "-e" args to ansible-playbook command
+
+# This is an account that must exist in OpenStack.
+# It is used to create projects, access, Heat templates
+admin_user: opentlc-mgr
+
+# Authenication credentials for OpenStack in order to create the things.
+# These should be included with your secrets, but are listed here for reference
+# osp_auth_url: 
+# osp_auth_username: 
+# osp_auth_password:
+# osp_auth_cloud:
+# osp_auth_project_domain: #usually set to "default"
+# osp_auth_user_domain: #usually set to "default"
+
+# The output_dir holds all of the files generated during the deployment
+# This includes generated Heat templates, SSH config, SSH keys
+output_dir: /tmp/output_dir
+
+# The name of the agnosticd config to deploy
+env_type: ocp4-disconnected-osp-lab
+
+# The {{ guid }} is used everywhere and it is what differentiates otherwise
+# identical environments. Make this unique. Usually they are 4 characters, but
+# it can be any reasonablre length.
+guid: mydefault
+
+# The name of the project that will be created in OpenStack for the user
+osp_project_name: "{{ guid }}-project"
+
+# The name of the OpenShift cluster that will be deployed.
+# This is primarily used if you want to automate the OpenShift deployment.
+cluster_name: cluster-{{ guid }}
+
+# Used to add metadata (tags) to OpenStack objects created
+project_tag: "{{ env_type }}-{{ guid }}"
+
+# Why is this config being deployed?
+# Some valid: development, ilt, production, event
+purpose: development
+
+# The type of cloud provider this will be deployed to
+cloud_provider: osp
+
+# This should be overwritten based on the user ordering the catalog item
+# It will be used by the bastion-student-user role and created on the bastion
+student_name: lab-user
+
+# Enable this if you want to create a user on the bastion
+# Mutually exclusive with {{ install_ipa_client }}
+install_student_user: true
+
+# Enable this if you want to use IPA for user authentication.
+# Mutually exclusive with {{ install_student_user }}
+install_ipa_client: false
+
+# TODO: What does this really do besides run the role?
+set_env_authorized_key: true
+env_authorized_key: "{{guid}}key"
+key_name: "default_key_name"
+
+# This is the user that Ansible will use to connect to the nodes it is
+# configuring from the admin/control host
+ansible_user: cloud-user
+remote_user: cloud-user
+
+# Run the bastion-lite role
+install_bastion: true
+
+# This should be obsolete and not required in the current form
+install_k8s_modules: false
+
+# FTL is used for grading and solving. It will pull in the external ftl-injector role.
+# This might be enabled when we have solvers to run or graders for ILT
+install_ftl: false
```

Comment:
I think we may want this to be true. That way we can add solve_lab scripts later for students to catch up.

Reply:
It is easy enough to override in the AgnosticV vars though, no? For the base config, I don't really need it or want it right now. I'm not passionate either way, though.